python/taichi/ui/gui.py

```diff
@@ -13,6 +14,21 @@ class Gui:
     def __init__(self, gui) -> None:
         self.gui = gui  #reference to a PyGui
 
+    @contextmanager
+    def subwindow(self, name, x, y, width, height):
+        """Creating a context manager for subwindow, all args are the same with `begin`.
+
+        Usage:
+
+        with gui.subwindow(name, x, y, width, height) as g:
```

Comment:
nit: One more indentation level, or sphinx can't recognize this as codeblock (IIUC): https://github.com/taichi-dev/taichi/blob/cc2dd3420f239c8811aa58f9dcfb89b9fbdfd8d3/python/taichi/lang/kernel_arguments.py#L94-L102

Reply:
Looks nice now！